CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this GitHub action will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
+### Added
+- Option to fail the status of the GitHub action if any alerts are found during the scan process. [#31](https://github.com/zaproxy/action-baseline/issues/31)
+
+### Changed
```

Comment:
Sorry for yet another comment, might worth saying that the action was changed to no longer fail by default on alerts.

Reply:
Done